docs/content/concepts/partitions-schedules-sensors/backfills.mdx

```diff
@@ -5,17 +5,17 @@ description: Dagster supports data backfills for each partition or subsets of pa
 
 # Backfills
 
-## Overview
+Backfilling is the process of running partitions for assets that either don’t exist or updating existing records. Dagster supports backfills for each partition or a subset of partitions.
 
-Dagster supports data backfills for each partition or subsets of partitions. After defining a [partitioned asset or job](/concepts/partitions-schedules-sensors/partitions), you can launch a _backfill_ which submits runs to fill in multiple partitions at the same time.
+After defining a [partitioned asset or job](/concepts/partitions-schedules-sensors/partitions), you can launch a backfill that will submit runs to fill in multiple partitions at the same time.
```

Comment:
similar to my suggestion [here](https://github.com/dagster-io/dagster/pull/16511#discussion_r1337808634), i wonder if we could simply say:
```suggestion
After defining a [partitions](/concepts/partitions-schedules-sensors/partitions), you can launch a backfill that will submit runs to fill in multiple partitions at the same time.
```